### DISCUSSION TITLE
Mobile viewport issues on pages with tables

### DISCUSSION THREAD

**Discussion by huge10** [filamentphp/filament#10422] [Help]
I have multiple resource pages with various tables (number of columns, length of column values). On a mobile viewport these tables become horizontally scrollable (as they should). However, on some the complete page has a horizontal scroll on mobile because somehow the table content pushes out the page width.

It seems to be caused by the table column content as removing some columns fixes the issue (while the table is still horizontally scrollable). However, I can't seem to pinpoint what is wrong with these columns (they are all text inputs, some with text others with numbers). The values don't seem to be longer then others.

Anyone else experiencing this? Any leads for me to investigate?

**Comment by kataklopgti** [comment]
same issue

**Comment by l-mendes** [comment]
I'm experiencing the same issue.
